app/assets/javascripts/3dbio_viewer/src/webapp/components/BasicInfoEntry.tsx

```diff
@@ -0,0 +1,137 @@
+                                                key={idx}
+                                                href={href}
+                                                target="_blank"
+                                                rel="noreferrer noopener"
+                                            >
+                                                {value}
+                                            </a>
+                                            {links.length != 1 && idx < links.length - 1 && ", "}
+                                        </React.Fragment>
+                                    );
+                                else if (itemToAdd)
+                                    return (
+                                        <React.Fragment key={idx}>
+                                            <span
+                                                className="anchor"
+                                                onClick={addOverlayItem(itemToAdd)}
+                                            >
+                                                {value}
+                                            </span>
+                                            {links.length != 1 && idx < links.length - 1 && ", "}
+                                        </React.Fragment>
+                                    );
+                            })}
+                        </span>
+                    )}
+                </li>
+            ))}
+        </ul>
+    );
+});
+
+function getItems(publication: PdbPublication) {
+    const items: Item[] = [
+        { name: i18n.t("Title"), value: publication.title },
+        { name: i18n.t("Abstract"), value: publication.abstract.unassigned },
+        { name: i18n.t("Authors"), value: publication.authors.join(", ") },
+        {
+            name: i18n.t("Journal"),
+            value: _.compact([
+                publication.journalInfo.isoAbbreviation,
+                publication.journalInfo.volume,
+                publication.journalInfo.year && "(" + publication.journalInfo.year + ")",
+            ]).join(" "),
+        },
+        {
+            name: i18n.t("Related entries"),
+            links: publication.relatedEntries.map(entry => ({
+                value: entry,
+                itemToAdd: buildDbItem(entry),
+            })),
+        },
+        {
+            name: i18n.t("PMID"),
+            links: publication.pubmedId
+                ? [
+                      {
+                          value: publication.pubmedId,
+                          href: "//europepmc.org/article/MED/" + publication.pubmedId,
+                      },
+                  ]
+                : undefined,
+        },
+        {
+            name: i18n.t("DOI"),
+            links: publication.doi
+                ? [
+                      {
+                          value: publication.doi,
+                          href: "//dx.doi.org/" + publication.doi,
```

Comment:
This url processing should be done before, when building the object, maybe adding `fieldUrl`